wordstats/cognate_user_evaluation.py

```diff
@@ -0,0 +1,32 @@
+from wordstats.cognate_info import CognateInfo
+from wordstats.getchunix import read_single_keypress
+from wordstats.edit_distance import LanguageAwareEditDistance
+
+# note: only executable in terminal, reactive to one-key stroke
+def evaluate_cognates(languageFrom, languageTo, method):
+    print(languageFrom, languageTo)
+    print("y:   whitelist")
+    print("q:   quit")
+    print("other:   blacklist")
+
+    cognateinfo = CognateInfo.load_cached(languageFrom, languageTo, method)
+    cognateinfo.save_candidates()
+    for key, values in cognateinfo.candidates.items():
+        for value in values:
+            if (key not in cognateinfo.blacklist.keys() or value not in cognateinfo.blacklist[key]) and\
+                (key not in cognateinfo.whitelist.keys() or value not in cognateinfo.whitelist[key]):
```

Comment:
can we move this complicated expression to it's own function? 
what i'd like a lot would be something like: `cognate.info.is_already_evaluated(key)`,
the fact that these two lines of code use four times cognateinfo should be a hint 
that this behavior belongs to that class.